packages/unmock-core/src/generator-utils.ts

```diff
@@ -0,0 +1,244 @@
+import {
+  Addl,
+  anyOfKeep,
+  anyOfReject,
+  Arr,
+  changeEnum,
+  changeListToTuple,
+  changeMaxItems,
+  changeMinItems,
+  changeRequiredStatus,
+  changeSingleSchema,
+  changeToConst,
+  header,
+  includeCodes,
+  MethodNames,
+  methodParameter,
+  oneOfKeep,
+  oneOfReject,
+  pathParameter,
+  removeCodes,
+  requestBody,
+  responseBody,
+} from "openapi-refinements";
+export {
+  Arr,
+  Addl,
+} from "openapi-refinements";
```

Comment:
Reaaally strong coupling to `openapi-refinements` and we don't yet know if that's useful for people as a separate library, including this in lerna would make development and maintenance faster in my opinion.